### PR TITLE
Replace viewpager android for react-native 0.61.0

### PR DESCRIPTION
**Pull request by jayshooo**
Since react-native version 0.61.0, ViewPagerAndroid is independent of ReactNative.

For this reason, the @ react-native-communty / viewpager dependency is required and the code needs to be modified.

Thank you for making good open source.

### CODE REVIEW

**Comment by ptomasroos** [comment]
Thank you! If someone could help review it would be great as I’m busy for two weeks with web projects.

And again. Big thanks.

Tomas

**Comment by heyman333** [comment]
I GOT ERROR 😩 
```
reuqireNativeCompoenent "RNCViewPager" was not found in the UIManager`
```

**Comment by jayshooo** [comment]
@heyman333 

meybe it would be help,

[https://github.com/react-native-community/react-native-webview/issues/534](https://github.com/react-native-community/react-native-webview/issues/534)

[https://github.com/react-native-community/react-native-viewpager/issues/18](https://github.com/react-native-community/react-native-viewpager/issues/18)

**Comment by heyman333** [comment]
@jayshooo THANK YOU!

Eventually, I think We need to install and link the `"@react-native-community/viewpager"` manullay 

I hope the above is added to the document

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
   },
   "homepage": "https://github.com/brentvatne/react-native-scrollable-tab-view#readme",
   "dependencies": {
+    "@react-native-community/viewpager": "^1.1.7",
```
Comment (eXist-FraGGer) [inline package.json]: Use 2.0.0 version

Comment (mistikk) [inline package.json]:
```suggestion
    "@react-native-community/viewpager": "^2.0.0",
```

**Comment by ptomasroos** [comment]
Waiting for these changes requested above ^ in order to merge.

**Comment by mlazowik** [comment]
If I may, I’d argue that bumping the version is an unrelated change, especially as it got released after this patch set was submitted.

**Comment by mlazowik** [comment]
Oh, and 2.0.1 got released ~20h ago. Requiring latest version could result in an endless chase IMO

File: package.json (hunk 1 above)
Comment (jayshooo) [reply package.json]: I changed version and updated PR. 
Thanks.

**Comment by mlazowik** [comment]
^ thank you! Though I don’t understand how this happened, the fork had public push rights?

**Comment by jayshooo** [comment]
> ^ thank you! Though I don’t understand how this happened, the fork had public push rights?

Alan Kim is my company accounts.

**Comment by mistikk** [comment]
image
@react-native-community/viewpager has issue with rn 0.61.2 :-( Please test on android, before merge it

**Comment by heyman333** [comment]
@mistikk 
https://github.com/ptomasroos/react-native-scrollable-tab-view/pull/1038#issuecomment-536943787

**Comment by mlazowik** [comment]
And now #1039 got merged with 1.1.7 :p

**Comment by MaximeConan** [comment]
There is an issue on Android with the last merge, you need to update the package.json with `"@react-native-community/viewpager": "^2.0.1"`.

I have try it on my project and it works fine 👍

**Comment by ptomasroos** [comment]
Can we rebase this in order to merge then?

**Comment by alexpchin** [comment]
@ptomasroos Are you able to merge this PR please?

**Comment by ptomasroos** [comment]
No the branch has conflicts. Waiting for it to get solved !

**Comment by die20** [comment]
@MaximeConan solution is the correct answer for RN 61.2

**Comment by jayshooo** [comment]
> No the branch has conflicts. Waiting for it to get solved !


I had fixed the crash. :)

**Comment by ptomasroos** [comment]
Thank you! Will release it

**Comment by NicolasSmit** [comment]
Guys, it was released without creating a new release :/ This is kind of breaking change

**Comment by testshallpasswork** [comment]
I don't think it was released. @ptomasroos Can you release it? It's a blocker for me.

**Comment by eBotti** [comment]
not really update released, please release it ....@ptomasroos

**Comment by ptomasroos** [comment]
Yes!

Will fix later tonight. 

Have a great day.

**Comment by EugeneDraitsev** [comment]
@ptomasroos sorry about rush, but maybe you have any updates about 0.11 release with these commits?

**Comment by ptomasroos** [comment]
Released as 1.0.0 breaking change

**Comment by SunflowerGJ** [comment]
I used the latest version1.0.0 in my new project, the error is still here🤦‍♀️，
image

**Comment by heyman333** [comment]
@SunflowerGJ

https://github.com/ptomasroos/react-native-scrollable-tab-view/pull/1038#issuecomment-536943787

could you try this?

**Comment by SunflowerGJ** [comment]
image
thanks，I forgot to link ,after linked ,It's well。but are you added "@react-native-community/viewpager" to your project's package.json? if I don't , I won't link successful
.

**Comment by SunflowerGJ** [comment]
> @SunflowerGJ
> 
> [#1038 (comment)](https://github.com/ptomasroos/react-native-scrollable-tab-view/pull/1038#issuecomment-536943787)
> 
> could you try this?

 thanks，I forgot to link ,after linked ,It's well。but are you  added "@react-native-community/viewpager" to your project's package.json?  if I don't , I won't link successful .
image

**Comment by ptomasroos** [comment]
Thanks for helping out @heyman333 !

**Pull request merged**